eng/common/scripts/ChangeLog-Operations.ps1

```diff
@@ -1,6 +1,10 @@
 # Common Changelog Operations
+. "${PSScriptRoot}\logging.ps1"
+. "${PSScriptRoot}\SemVer.ps1"
 
 $RELEASE_TITLE_REGEX = "(?<releaseNoteTitle>^\#+.*(?<version>\b\d+\.\d+\.\d+([^0-9\s][^\s:]+)?)(\s(?<releaseStatus>\(Unreleased\)|\(\d{4}-\d{2}-\d{2}\)))?)"
+$UNRELEASED_TAG = "(Unreleased)"
+$DATE_FORMAT = "yyyy-MM-dd"
```

Comment:
Given this will be at the global variable scoped we should update the names to be a little more specific. Maybe something like "CHANGELOG_DATE_FORMAT" and "CHANGELOG_UNRELEASED_STATUS"